internal/packaging/dependency/mongo.go

```diff
@@ -11,7 +11,7 @@ import (
 
 // InstallMongo installs a mongo server for juju from snap using the specified
 // snap channel.
-func InstallMongo(snapChannel string) error {
+func InstallMongo() error {
 	snapManager := manager.NewSnapPackageManager()
-	return snapManager.Install(fmt.Sprintf("juju-db --channel %s", snapChannel))
+	return snapManager.Install(fmt.Sprintf("juju-db"))
```

Comment:
Here we probably want `"juju-db --channel 4.4/stable"` right?

`latest/stable` is 4.0

Reply:
I actually don't care, but yeah, I'll set that as hard-coded.